packages/backend/src/open_payments/wallet_address/service.ts

```diff
@@ -168,6 +169,12 @@ async function createWalletAddress(
   }
 
   try {
+    const tenantId = options.tenantId
+      ? options.tenantId
+      : deps.config.operatorTenantId
+    const asset = await deps.assetService.get(options.assetId, tenantId)
```

Comment:
I don't think the service needs to know/use `config.operatorTenantId`, it'll get it from the calling service if needs to

Reply:
I think this was just an old commit.

Reply:
No, it has been updated already.